modules/release-notes/snip-deprecated-functionality-rhidp-1138.adoc

```diff
@@ -0,0 +1,58 @@
+[id="deprecated-functionality-rhidp-1138"]
+= `spec.application.image`, `spec.application.replicas` and `spec.application.imagePullSecrets` fields are deprecated
```

Comment:
That would be less confusing to customers, IMO. If I read something as a breaking change, it would mean that I absolutely need to update my settings prior to upgrading, which is not the case here.